Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -78,6 +79,33 @@ class FocusableWindowsTemplate {
      */
     disableSystemFocusVisuals: PropTypes.bool,
 
+    /**
+     * Controls how view is important for accessibility which is if it
+     * fires accessibility events and if it is reported to accessibility services
+     * that query the screen. For more details refer to Libraries\Components\View\ViewPropTypes.js
+     */
+    importantForAccessibility: PropTypes.oneOf([
+      'auto',
+      'yes',
+      'no',
+      'no-hide-descendants'
+    ]),
+
+    /**
+     * Provides additional traits to screen reader. By default no traits are
+     * provided unless specified otherwise in element. For more details refer to Libraries\Components\View\ViewPropTypes.js.
+     */
+    accessibilityTraits: PropTypes.oneOfType([
+      PropTypes.oneOf(AccessibilityTraits),
+      PropTypes.arrayOf(PropTypes.oneOf(AccessibilityTraits)),
+    ]),
+
+    /**
+     * When `accessible` is true, the system will try to invoke this function
```

Comment:
What is 'accessible' ?

Reply:
copied comment, fixed.


---
In reply to: [173052410](https://github.com/Microsoft/react-native-windows/pull/1698#discussion_r173052410) [](ancestors = 173052410)